datumaro/components/visualizer.py

```diff
@@ -118,22 +150,122 @@ def vis_one_sample(
             else None
         )
 
+        context = defaultdict(list)
         for ann in annotations:
             if ann.type in self.ignored_types:
                 warnings.warn(f"{ann.type} in self.ignored_types. Skip it.")
                 continue
 
-            if ann.type == AnnotationType.bbox:
-                self._draw_bbox(ann, label_categories, ax)
+            if ann.type in self._draw_func:
+                self._draw_func[ann.type](ann, label_categories, fig, ax, context[ann.type])
```

Comment:
why do we need to get context? could you provide an example for context?

Reply:
`context` includes previously drawing history for each annotation type. It is necessary to avoid drawing again over an already drawn place. For example, multi label dataset has multiple `Label`s in a `DatasetItem`. If we don't keep `context`, it has to draw `Label` at the same place again and again. In addition to `Label`, `Caption` also requires `context` for now.

For example, 
![image](https://user-images.githubusercontent.com/26541465/197707212-4bd318b8-170e-41c6-b161-236ab9dd3e64.png)